blog/2019-05-29-gsoc-2019-an-overview/index.md

```diff
@@ -30,7 +30,7 @@ Game modes bundle these modules together enabling players to immerse themselves
 
 The idea for the [Light and Shadow] game mode dates back to 2013 when SuperSnark came up with neat artworks backed by elaborate lore of quarreling factions and way too many ideas for a single person to implement.
 
-![LightAndShadow](./las_red-keep.png)
+![LightAndShadow](./las_red-keep.jpg)
```

Comment:
I'm not sure about the quality diff here :thinking: also, looks quite neat with the transparent background...